docs/source/operators/deploy-kubernetes.md

```diff
@@ -15,248 +17,495 @@ The following sample kernel specifications apply to Kubernetes deployments:
+- Tipically this will be done by Cluster Admininstrator.
 
-The service is currently configured as type `NodePort` but is intended for type `LoadBalancer` when appropriate network plugins are available. Because kernels are stateful, the service is also configured with a `sessionAffinity` of `ClientIP`. As a result, kernel creation requests will be routed to different deployment instances (see deployment) thereby diminishing the need for a `LoadBalancer` type. Here's the service yaml entry from [enterprise-gateway.yaml](https://github.com/jupyter-server/enterprise_gateway/blob/main/etc/kubernetes/enterprise-gateway.yaml) (for helm, see [service.yaml](https://github.com/jupyter-server/enterprise_gateway/blob/main/etc/kubernetes/helm/enterprise-gateway/templates/service.yaml)):
+Create `values-kip.yaml` file with the following content:
 
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app: enterprise-gateway
-  name: enterprise-gateway
-  namespace: enterprise-gateway
-spec:
+```bash
+global:
+  # Craete RBAC resources
+  rbac: false
+
+deployment:
+  enabled: false
+
+ingress:
+  enabled: false
+
+# Kernel Image Puller (daemonset)
+kip:
+  enabled: true
+  serviceAccountName: 'kernel-image-puller-sa'
+  podSecurityPolicy:
+    create: false
+  resources:
+    limits:
+      cpu: 100m
+      memory: 200Mi
+    requests:
+      cpu: 50m
+      memory: 100Mi
+```
+
+Run helm deploy:
+
+```bash
+helm  upgrade --install \
+  jupyter-e-gw-rbac \
+  jupyter-server/enterprise_gateway/etc/kubernetes/helm/enterprise-gateway \
+   --kube-context [mycluster-context-name] \
+   --namespace [namespace-name] \
+   -f values-kip.yaml
+```
+
+###### 3. Helm release to deploy Enterprise Gateway.
+
+- This can be done by namespace Administrator.
+
+Create `values-eg.yaml` file with the following content:
+
+```bash
+global:
+  rbac: false
+
+service:
+  type: "ClusterIP"
   ports:
-    - name: http
+    # The primary port on which Enterprise Gateway is servicing requests.
+    - name: "http"
       port: 8888
       targetPort: 8888
-    - name: response
+      # nodePort: 32652 # optional nodePort
+    # The  port on which Enterprise Gateway will receive kernel connection info responses.
+    - name: "http-response"
       port: 8877
       targetPort: 8877
-  selector:
-    gateway-selector: enterprise-gateway
-  sessionAffinity: ClientIP
-  type: NodePort
+      # nodePort: 30481 # optional nodePort
+
+deployment:
+  enabled: true
+  resources:
+    limits:
+      cpu: 2
+      memory: 10Gi
+    requests:
+      cpu: 500m
+      memory: 2Gi
+  # Update to deploy multiple replicas of EG.
+  replicas: 1
+
+ingress:
+  enabled: true
+  # Ingress resource host
+  hostName: "jupiter-e-gw.example.com"
```

Comment:
Does this value need a placeholder?  Let's not persist this value as it's listed here.